tests/IntegrationTests/EndToEndTests/E2E_UploadFileToMeshStorage.cs

```diff
@@ -0,0 +1,85 @@
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using IntegrationTests.Helpers;
+using System.Net;
+using System.Net.Http.Headers;
+
+namespace Tests.Integration.EndToEndTests
+{
+    [TestClass]
+    [TestCategory("Integration")]
+    public class E2E_UploadFileToMeshStorage : BaseIntegrationTest
+    {
+        private BlobStorageHelper _blobStorageHelper;
+        private string _blobContainerName;
+        private string _episodeFilePath;
+        private string _episodeFileName;
+        private string _participantFilePath;
+        private string _participantFileName;
+        private ILogger<E2E_UploadFileToMeshStorage> _logger;
+        private MeshMailboxHelper _meshMailboxHelper;
+
+        [TestInitialize]
+        public async Task TestInitialize()
+        {
+            _blobStorageHelper = ServiceProvider.GetService<BlobStorageHelper>();
+            _blobContainerName = AppSettings.BlobContainerName;
+            _logger = ServiceProvider.GetService<ILogger<E2E_UploadFileToMeshStorage>>();
+            _meshMailboxHelper = new MeshMailboxHelper(AppSettings.Endpoints.MeshSandboxOutput, AppSettings, HttpClient);
+
+            _episodeFilePath = AppSettings.FilePaths.LocalEpisodesCSVFile;
+            _participantFilePath = AppSettings.FilePaths.LocalSubjectsCSVFile;
+
+            var episodeFileNameIndex = _episodeFilePath.LastIndexOf('/');
+            _episodeFileName = _episodeFilePath.Substring(episodeFileNameIndex+1);
+
+            var participantFileNameIndex = _participantFilePath.LastIndexOf('/');
+            _participantFileName = _participantFilePath.Substring(participantFileNameIndex+1);
+
+            // Ensure all config is set
+            AssertAllConfigurations();
+
+            // Get the direct filepath to the csv test file and ensure it exists
+            var localFilePath = Path.Combine(AppContext.BaseDirectory, AppSettings.FilePaths.LocalEpisodesCSVFile);
+            Assert.IsTrue(localFilePath.Length > 0);
```

Comment:
`Assert.IsTrue(localFilePath.Length > 0);` Just thinking, this seems to only check that the file path string is not empty, I wonder if we can update the assertion to check if the file actually exists e.g `Assert.IsTrue(File.Exists(localFilePath), $"File does not exist at path: {localFilePath}");`

Reply:
Resolved using suggestion